### PR TITLE
added cmodules for clean architecture

### PR DESCRIPTION
**Pull request by benjaglobant**
PR created to add the base structure of the clean architecture that will be used in the "HarryPotterApp"
 - "data" module created (Android Module)
 - "di" module created (Android Module)
 - "domain" module created (Kotlin Module)
 - Also I put the dependencies between those modules into each gradle file.
Modules image:
image

### CODE REVIEW

File: data/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mauroLonghi) [inline data/src/main/AndroidManifest.xml]: Pls, if you not use the xmls:android , remove it

File: di/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mauroLonghi) [inline di/src/main/AndroidManifest.xml]: Same, remove the label

File: data/src/main/AndroidManifest.xml (hunk 1 above)
Comment (benjaglobant) [reply data/src/main/AndroidManifest.xml]: Done!

File: di/src/main/AndroidManifest.xml (hunk 2 above)
Comment (benjaglobant) [reply di/src/main/AndroidManifest.xml]: Done!

**Review by mauroLonghi — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by hernanhorquin96 — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**